### PR TITLE
No 3rd-party domains

### PR DESCRIPTION
**Pull request by eatonphil**
Noticed some bugs coming back that zxcvbn wasn't defined. No need to go out to third-party domains for those. This switches them (zxcvbn and font awesome) to being served by us.

Things to check:
* weblish
* password strength meter
* icons

### CODE REVIEW

**Comment by eatonphil** [comment]
I'll add in the changelog and versioning changes once https://github.com/linode/manager/pull/2293 gets merged.

File: package.json [hunk 1]
```diff
@@ -124,6 +126,6 @@
     "redux": "^3.0.5",
     "redux-thunk": "^2.1.0",
     "xterm": "^2.0.1",
-    "zxcvbn": "^4.4.1"
+    "zxcvbn": "https://registry.npmjs.org/zxcvbn/-/zxcvbn-4.4.2.tgz"
```
Comment (mparke) [inline package.json]: I think these registry urls is a bug, if you rm node_modules and reinstall it should save just the version correctly

Comment (eatonphil) [inline package.json]: kk, will try

Comment (eatonphil) [inline package.json]: Fixed.

**Comment by mparke** [comment]
@na3d

**Comment by eatonphil** [comment]
thanks!

**Pull request merged**